### DISCUSSION TITLE
Can we have these boxes clickable ? not just when it was clicked on the hyper-text？ entire box? Thank you.

### DISCUSSION THREAD

**Discussion by mobilelifeful** [hbstack/.github#25] [Ideas]
Each box, like color and  Header Themes..... Thank you.

![image](https://github.com/hbstack/.github/assets/91811647/26982325-4e1b-4fad-9f53-7f5b4df9dd02)

**Comment by razonyang** [comment]
Do you mean the article cards?

![image](https://github.com/hbstack/.github/assets/17720932/9598b1a2-590c-4c35-b69c-dc5edc00807a)

**Comment by razonyang** [comment]
Since `github.com/hugomods/bootstrap` `v0.11.1`, you can specify the `linkText` as an empty string, then the card will be clickable.

```sh
hugo mod get github.com/hugomods/bootstrap@v0.11.1
```

See also

- https://hugomods.com/en/bootstrap/icon-grid/
- https://hugomods.com/en/bootstrap/article-cards/
